src/turnkeyml/cli/report.py

```diff
@@ -88,8 +88,8 @@ def summary_spreadsheets(args) -> None:
                                     or fs.Keys.BENCHMARK_STATUS
                                 )
                                 or fs.Keys.STAGE_STATUS in key
-                            ) and value == bd.FunctionStatus.INCOMPLETE.value:
-                                value = bd.FunctionStatus.KILLED.value
+                            ) and value == bd.FunctionStatus.INCOMPLETE:
+                                value = bd.FunctionStatus.KILLED
```

Comment:
Nit: turnkeyml.common.build is imported as build in all other files
Ideally prefer the same import for coherence.

Reply:
done!